examples/dreamcast/library/exports.txt

```diff
@@ -0,0 +1,7 @@
+include kos/exports.h
+include string.h
+
+# Additional newlib functions to export
+# Some part of it already exported by kernel/exports.txt
+strcmp
```

Comment:
Why would it be that we add exports for an individual library? Shouldn't the exports reflect everything that KOS is guaranteed to offer? It seems to me (though I may misunderstand) that if KOS is not guaranteed to export these, then we should be linking libc to the library to guarantee they are present. If KOS *is* guaranteed to export these then they should be listed in the kernel/exports.

Reply:
KOS is guaranteed to export only some of the libc functions that it uses. You don't need to export everything at once, since they will all be linked into the binary, regardless of whether it is used or not. To minimize memory consumption, you need to select only the necessary functions. And this is individual for each project.
Although if someone is not lazy and writes an export of all libc functions, this can be included as an option. Like exporting the entire libc library. But this is only good for testing, for real projects I think it would be wasteful.

Reply:
Ah ok, that makes sense. Would that same principle also follow for the rest of the KOS API then? Rather than always exporting as much as we can of the API?